db/schema.rb

```diff
@@ -486,7 +486,6 @@
     t.index ["exclude_article_ids"], name: "index_display_ads_on_exclude_article_ids", using: :gin
     t.index ["placement_area"], name: "index_display_ads_on_placement_area"
     t.index ["target_geolocations"], name: "gist_index_display_ads_on_target_geolocations", using: :gist
-    t.index ["target_geolocations"], name: "index_display_ads_on_target_geolocations"
```

Comment:
ack I removed the ineffective b-tree index (discussed in https://github.com/forem/forem/pull/19834#discussion_r1276559500) but i did not commit the schema